### PR TITLE
NewsWeek web page clone

### PR DESCRIPTION
**Pull request by rukundoeric**
### What does this do? 
* Build Responsive Design with Bootstrap
### Description of Task to be completed? 
* A structural clone of  The NewsWeek web page
### How should this be manually tested? 
* Clone this repo  https://github.com/bini-i/Newsweek-Clone/tree/bootstrap-feature-branch
* Navigate to project directory.
* If you have a live server in you vs-code, just run it.
* Otherwise just open the index page with your browser.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1744 @@
...
+          </div>
+          <div class="d-flex">
+            <a class="navbar-brand" href="#"
+              ><img id="newsweek-logo" class="logo" src="./img/logo.svg" alt="logo"
+            /></a>
+          </div>
+          <div class="d-flex align-items-center links">
+            <a class="login" href="#">LOGIN</a>
+            <a href="" class="mr-3 user"><i class="fas fa-user"></i></a>
+            <a href="" class="d-flex align-items-center subscribe-container">
```
Comment (arslanbisharat) [inline index.html]: use # as a path

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

 **Repo**

- [ ]Remove !important from CSS file 

- [x] Add something about your project in about section of your repo. This can help to understand features in your repo. 

- [x] Please use less CSS because the purpose of this project is to master bootstrap.

- [x] For href tag you can use the path **#**

- [x] Please make your repo more professional by adding all styling in one folder like screenshots, CSS, sass, images folder.  

**main**

- [x] On medium size, the text is overflowing. 
chrome-capture (21)


- [x] Here is an attachment from the original page. The images have more height than the text to the right. 
chrome-capture (17)

- [x] You have copy-pasted the same code again and again. Please try to make it more attractive by changing images and using similar titles as presents on the original page. 
chrome-capture (16)

- [x] You can have a look at the bottom attachment. at small size object Iphone5 there is white space but on the original page, this didn't happen. 
chrome-capture (18)


**Footer**
- [x]  Footer is disorganized at small objects.

- [x]  Please make footer links as [present on the original page](https://www.newsweek.com/). 
chrome-capture (15)

- [x] Item are not align as present on the original page. Please make it as present on the original 
chrome-capture (15)
page. 

**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello team!! Great job working on the previously required changes!

## Required Changes
### [Mandatory] :red_circle: 

- [x] Add to ```.gitignore``` files unrelated to the project, and nonproduction ready. The prettier config file is for local development, and SCSS uses the final CSS process file.
image

- [x] MIT link is broken in README

- [x] Give all image ```alt="``` attribute an alternative text. Don't leave it blank.

- [x] Navbar is missing elements
image

- [x] One element should be different from the others.
image

- [x] Element does not match original in tablet view
image

- [x] Missing space between elements in subscribe section
image

- [x] Elemens don't match in newsletter section
image

- [x] The order of the sections on the mobile version doesn't match the actual page. 
**Original**
- top story
- featured stories
- The debate
- Opinion
- More Stories
- My turn

**Clone**
- **Features stories**
- **Top stories**
- **My turn**
- More stories
- the debate
- opinion

- [x] My turn section doesn't match original on the mobile version.
image

- [x] Subscribe section doesn't look as original on the mobile version
image

- [x] Newsletter section doesn't look as the original on the mobile version
image

- [x] Footer doesn't look like the original on the mobile version
image

- [x] Navbar doesn't look like the original on the mobile version
image

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: Hello!! Great job working on the previously required changes!

### [Mandatory] :red_circle: 

- [x] Elemens don't match in newsletter section
>   image
- [x]  The order of the sections on the mobile version doesn't match the actual page.
>   **Original**
> * top story
> * featured stories
> * The debate
> * Opinion
> * More Stories
> * My turn
> 
> **Clone**
> 
> * **Features stories**
> * **Top stories**
> * **My turn**
> * More stories
> * the debate
> * opinion

- [x] Navbar is missing menu button in mobile
image

- [x] Newsletter section doesn't look as the original on the mobile version
>   image

**Original**
image

**Clone**
image

### [Comments] :green_circle: 
- A great tool to test if the project looks as intended in different screen sizes is using the browser media query inspector.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Great job guys with all the implemented changes! 👏 
Your project looks pretty nice! 💯 
Just a few adjustments to get it approved
Kindly, check my comments below 👇  apply the changes and resubmit your project for a new code review

## Mandatory Changes
- Kindly remove some CSS lines, this is a project to master bootstrap and you should use it everywhere it is available. You can display flex with d-flex, display none with d-none, etc... you do not need those lines in your CSS file, please implement everything you can using bootstrap classes.
- Screens below 768px:
   - The sections are not in the right order after opinion comes 'my turn' and then 'culture&travel'. Please modify it to make it equal to the original
   - In the magazine section, the first image should be bigger than the rest, kindly modify it
   - In the editor's picks section, next to the title should be the logo of the publication, now it has a photo, kindly remove it and add the logo
   - In the US, Business, sport.... sections, make the aside items pictures wider to look like the original
   - In the subscribe section, the images test and button must be aligned centre
   - in the footer, the social media icons should go under the Newsweek letters and copyright, please make it happen.

**Comment by bini-i** [comment]
Fix Order of myturn and culture section …
Fix magazine section, first image made larger
Fix editors section logo display
Fix US, Business, sport section picture made wider
Fix subscribe section, image made center
Fix footer section social media order in mobile view

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,730 @@
...
+  width: 100%; }
+  @media (min-width: 768px) and (max-width: 991px) {
+    .line {
+      display: none; } }
+  @media (max-width: 767px) {
+    .line {
+      display: none; } }
+
+.container {
+  padding: 0 !important;
```
Comment (umairarshadbutt) [inline assets/css/style.css:42]: Remove `!important`

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,730 @@
...
+      max-width: 100% !important;
+      padding: 0 !important; } }
+  @media (max-width: 767px) {
+    .container {
+      max-width: 100% !important;
+      padding: 0 !important; } }
+  @media (max-width: 480px) {
+    .container {
+      max-width: 100% !important;
+      padding: 0 !important; } }
```
Comment (umairarshadbutt) [inline assets/css/style.css:58]: Remove `!important`

File: assets/css/style.css [hunk 4]
```diff
@@ -0,0 +1,730 @@
...
+  padding: 0 !important;
+  background-color: #fff; }
+  @media (min-width: 992px) {
+    .container {
+      max-width: 1310px !important; } }
+  @media (min-width: 768px) and (max-width: 991px) {
+    .container {
+      max-width: 100% !important;
+      padding: 0 !important; } }
+  @media (max-width: 767px) {
```
Comment (umairarshadbutt) [inline assets/css/style.css:51]: Remove `!important`

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly and work on the suggested changes  👇.
- [ ] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)


- **When you have made the specified changes, please submit another code review. Thank you!**


**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by rukundoeric** [comment]
Thanks, 

* `!Important` is removed everywhere in the code.

**Review by arslanbisharat — APPROVED** [review]
Congratslutons Approved 🎉
Hello Developer 👋
You are on 🔥
Thank you so much for making those changes and I am very happy to approve the project. You have done a great job. 👏
Please keep working hard and move forward with your curriculum.
Looking forward to seeing your next project. 😄
---------------------------------------------------------------------------------------------------------------
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**